packages/app/lib/ui/widgets/image_carousel.dart

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import 'package:flutter/material.dart';
+import 'package:carousel_slider/carousel_slider.dart';
+
+import 'package:app/ui/colors.dart';
+
+class ImageCarousel extends StatelessWidget {
+  final List<Image> images;
+  final Function? onDelete;
+
+  ImageCarousel({super.key, required this.images, this.onDelete});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.symmetric(vertical: 5),
+      child: CarouselSlider(
```

Comment:
I would somehow assume that the items "within" the box would move, not the whole box itself when I slide from one to the next image. Would it be possible to have the border around the carousel instead?